File: tiga.php
<?php

$barang = "99.900";
$hari = 10%
$diskon = 5%
$produk = 55;
$hasil1 = ($diskon/100)*$produk*$barang;
$hasil2 = ($hari/100);
$jumlah_barang = $_POST['value'];

if (empty($_POST['value'])) {
    echo '0';
}

echo "
harga barang : $barang <br/>
diskon jika membeli lebih dari 50 barang : $diskon <br/>
diskon jika membeli pada hari senin/kamis : $hari <br/>
";

if ($barang >=50 AND $diskon = $hasil1) {
    echo "maka total belanjaan ($hasil1)<br/>";
}
else if ($hari AND $barang) {
    echo "Jika anda membeli pada hari senin/kamis makan akan mendapatkan diskon 10% <br/>";
}
else {
    echo "kamu tidak mendapatkan diskon";
}

?>
<form method="post" action="">
    <input type="text" name="value" placeholder="Jumlah barang yang dibeli">
    <input type="submit">
</form>

File: tiga.py
harga_barang = 99900
print("Harga barang:", harga_barang)
jumlah_barang = int(input("Jumlah barang yang dibeli: "))

if (jumlah_barang < 50) :
    total = harga_barang * jumlah_barang
    print("Total harga barang:", total)
elif (jumlah_barang >= 50) :
    hari = str(input("Hari membeli: "))
    if (hari == "Senin","senin","Kamis","kamis") :
        total_diskon = 10 + 5
        total = total_diskon/100*(harga_barang * jumlah_barang)
        print("Kamu mendapat diskon 5% dan mendapat diskon tambahan 10%")
        print("Total harga barang:", total)
    else :
        total = 5/100*(harga_barang * jumlah_barang)
        print("Kamu mendapat diskon 5%")
        print("Total harga barang:", total)
else :
    print("Masukan Data dengan benar")